rdkit/Chem/MolDb/Loader_orig.py

```diff
@@ -158,7 +158,7 @@ def LoadDb(suppl, dbName, nameProp='_Name', nameCol='compound_id', silent=False,
       pass
     curs.execute(f'create table {regName} ({",".join(typs)})')
   else:
-    curs.execute(f'select * from {(regName, )} limit 1')
+    curs.execute(f'select * from {regName} limit 1')
```

Comment:
This change is not identical:
```
>>> regName = 'abc'
>>> f'select * from {(regName, )} limit 1'
"select * from ('abc',) limit 1"
>>> f'select * from {regName} limit 1'
'select * from abc limit 1'
```

Reply:
I have tested on GDB online

```python
import sys
import os

print(sys.version, sys.version_info)

regName = 'abc'

x = 'select * from %s limit 1' % (regName, )
print(x)

y = f'select * from {regName} limit 1'
print(y)
```

The result is `x==y` is evaluate to be True
Tested on GDB online and `online-python.com`

You can see the previous commit. It raised error in unit test, and I have to fix twice of it.